docker-images/build.sh

```diff
@@ -37,12 +39,124 @@ function alternate_base {
     fi
 }
 
+function build_kafka_images {
+
+    local tag=$1
+    local java_version=$2
+    local targets=${@:3}
+
+    echo -e "\n\nDebug:\n"
```

Comment:
This is probably not necessary once it is working ;-)

Reply:
Sorry, left that in by mistake.